apps/main/src/loan/components/PageCrvUsdStaking/Statistics/DistributionsBarChart.tsx

```diff
@@ -0,0 +1,72 @@
+import type { ScrvUsdRevenue } from '@/loan/entities/scrvusdRevenue'
+import { BarChart, Bar, Cell, XAxis, YAxis, CartesianGrid, Tooltip, ResponsiveContainer } from 'recharts'
+import { formatNumber, formatDate } from '@ui/utils/utilsFormat'
+import { useTheme } from '@mui/material/styles'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { toDate } from '@curvefi/prices-api/timestamp'
+import { Stack, Box } from '@mui/material'
+import DistributionsChartTooltip from './DistributionsChartTooltip'
+
+const { FontSize } = SizesAndSpaces
+
+type RevenueDistributionsBarChartProps = {
+  data: ScrvUsdRevenue | null
+  height?: number
+}
+
+const RevenueDistributionsBarChart: React.FC<RevenueDistributionsBarChartProps> = ({ data, height = 400 }) => {
+  const {
+    design: { Color, Text },
+  } = useTheme()
+  const gridLineColor = Color.Neutral[300]
+  const gridTextColor = Text.TextColors.Tertiary
+  const barColor = Color.Secondary[500]
+
+  return (
+    <Stack direction="column" spacing={0} width="100%" sx={{ position: 'relative' }}>
```

Comment:
Aren't these the defaults? :thinking: 

```suggestion
    <Stack sx={{ position: 'relative' }}>
```

Reply:
yea correct, these are defaults, removing